src/Templates/TodoTemplate/Bit.TodoTemplate/Api/Extensions/ODataOperationFilter.cs

```diff
@@ -100,5 +104,10 @@ public void Apply(OpenApiOperation operation, OperationFilterContext context)
                 Required = false
             });
         }
+
+        if (odataQueryOptionsParameter is not null)
+        {
+            operation.Parameters.Remove(operation.Parameters.Single(p => p.Name == odataQueryOptionsParameter.Name));
+        }
```

Comment:
what is this?
please remove these changes from this PR.

Reply:
These changes make Swagger's UI compatible with APIs that have Odata input.
Do you think these changes should happen in the form of a new issue?

Reply:
I remove these changes and register another issue for it. @msynk